integration-tests/test_checkin.py

```diff
@@ -5,17 +5,16 @@
 
 pytestmark = pytest.mark.usefixtures("register_subman")
 
-HOST_DETAILS = "/var/lib/insights/host-details.json"
 
-
-def test_ultralight_checkin(insights_client):
+def test_ultralight_checkin(insights_client, test_config):
     """test --checkin
     Sends nothing but canonical facts to update the host stale and updated timestamp
     """
     insights_client.register()
-    # Performing check-results operation provides latest host data in host-details.json
+    # Performing check-results operation provi des latest host data in host-details.json
     insights_client.run("--check-results")
-    with open(HOST_DETAILS, "r") as data_file:
+    host_details = test_config.get("insights", "host_details")
```

Comment:
In my opinion this var should either go to constants.py or let it be the way it is right now. this is not a configuration option that could go in config file.

Reply:
updated to your request.